exercises/chapter12/test/Copy.purs

```diff
@@ -0,0 +1,23 @@
+module Test.Copy where
+
+import Prelude
+import Data.Either (Either(..))
+import Effect.Aff (Aff, attempt, message)
+import Effect.Class.Console (log)
+import Node.Encoding (Encoding(..))
+import Node.FS.Aff (readTextFile, writeTextFile)
+import Node.Path (FilePath)
+
+copyFile :: FilePath -> FilePath -> Aff Unit
+copyFile file1 file2 = do
+  my_data <- readTextFile UTF8 file1
+  writeTextFile UTF8 file2 my_data
```

Comment:
I wanted to mirror the style of the JS version, which is the preceding example in the text. It would be clearer where these code snippets are used if I setup mdbook anchors.
``` js
async function copyFile(file1, file2) {
  var fs = require('fs');
  let data = await fs.readFile(file1, { encoding: 'utf-8' })
  fs.writeFile(file2, data, { encoding: 'utf-8' })
}
```
Although now that I pasted this, I realize it may be even clearer to move the `require` up a level.

Reply:
Makes sense.

Yeah, moving that `require` up would make it mirror things better.